R/cdfPlot.R

```diff
@@ -113,6 +113,8 @@ cdfPlot <- function(contrastDF,
 
     assertthat::assert_that(pvalCol %in% colnames(contrastDF),
                             msg = "Specified pvalCol not found in the supplied dataframe (contrastDF).")
+    assertthat::assert_that(plotType %in% c("ggplot", "canvasXpress"),
+                            msg = "Plot type must be either ggplot or canvasXpress.")
```

Comment:
since default is CX list that option first.

Reply:
Done